### DISCUSSION TITLE
data/2018/01/23/opencv-1/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#147] [Comments]
# data/2018/01/23/opencv-1/

OpenCV Python으로 이미지/비디오 읽기! OpenCV를 이용해 이미지 파일을 읽고 보고 저장하는 방법에 대해 알아보겠습니다 키워드 : cv2.imread(), cv2.imshow(), cv2.imwrite(), cv2.VideoCapture(), cv2.VideoWriter() 

https://zzsza.github.io/data/2018/01/23/opencv-1/

**Comment by bemoregt** [comment]
비디오 뿐 아니라  animatedGIF 파일도 억세스 가능합니다.

GPTs 에 OpenCV 에 특화된 챗봇(chatGPT4)을 만들었습니다.
https://chat.openai.com/g/g-6w1h9ncEg-opencvgpt

들어가서 아무질문이나 던져보세요~

좋아요 드립니다.
